homeassistant/components/openai_conversation/strings.json

```diff
@@ -106,6 +110,16 @@
       }
     }
   },
+  "issues": {
+    "organization_verification_required": {
+      "title": "Organization verification required",
+      "description": "Your organization must be verified to use the model `gpt-image-1`. Please go to: https://platform.openai.com/settings/organization/general and click on Verify Organization. If you just verified, it can take up to 15 minutes for access to propagate."
```

Comment:
The URL should be injected from the integration as a translation placeholder

Reply:
Done